### PR TITLE
[rook/ceph] set request parameters for the pods

### PR DESCRIPTION
**Pull request by llamerada-jp**
Signed-off-by: Yuji Ito <llamerada.jp@gmail.com>

### CODE REVIEW

File: rook/base/ceph-hdd/cluster.yaml [hunk 1]
```diff
@@ -37,11 +68,12 @@ spec:
     storageClassDeviceSets:
       - name: set1
         count: 18
-        tuneDeviceClass: true
+        ## send bug report https://github.com/rook/rook/issues/6789
         resources:
           requests:
             cpu: "500m"
-            memory: "4Gi"
```
Comment (binoue) [inline rook/base/ceph-hdd/cluster.yaml:44]: Why did you reduce the request memory?
Based on [the Ceph document](https://docs.ceph.com/en/latest/start/hardware-recommendations/#memory), osd may result in degraded performance with between 2GB and 4GB memory.

File: rook/base/ceph-ssd/cluster.yaml [hunk 2]
```diff
@@ -7,6 +7,37 @@ metadata:
...
   dataDirHostPath: /var/lib/rook
+  resources:
+    mgr:
+      requests:
+        cpu: "500m"
+        memory: "512Mi"
+    mon:
+      requests:
+        cpu: "500m"
+        memory: "1Gi"
```
Comment (ymmt2005) [inline rook/base/ceph-ssd/cluster.yaml:18]: Does `mon` really consume memory more than 1GiB?

Comment (toshipp) [reply rook/base/ceph-ssd/cluster.yaml:18]: I don't know that, but rook recommends it.
https://github.com/rook/rook/blob/master/Documentation/ceph-cluster-crd.md#cluster-wide-resources-configuration-settings

File: rook/base/ceph-hdd/cluster.yaml (hunk 1 above)
Comment (toshipp) [reply rook/base/ceph-hdd/cluster.yaml:44]: Yes, rook document says 2G is the minimum requirement.
If OSD requires more memory, this is a request, so OSD can consume more memory.
This is a baseline, we would tune the value later.

File: rook/base/ceph-ssd/cluster.yaml (hunk 2 above)
Comment (ymmt2005) [reply rook/base/ceph-ssd/cluster.yaml:18]: I see.

**Review by toshipp — APPROVED** [review]
LGTM

**Pull request merged**